tendrl/node_agent/manager/pull_hardware_inventory.py

```diff
@@ -122,6 +125,52 @@ def getTendrlContext():
     return tendrl_context
 
 
+def get_node_network():
+    '''returns structure
+
+    {"interface_name": {"ipv4": ["ipv4address", ...],
+                        "ipv6": ["ipv6address", ...],
+                        "netmask": ["subnet", ...]
+                        "status":"up/down"}, ...}
+    '''
+    interfaces = ni.interfaces()
+    rv = {}
+    for interface in interfaces:
+        if interface not in ['lo', 'virbr0', 'virbr0-nic']:
```

Comment:
Define a constant to store the list of interfaces which are not to be considered.